GoogleForCreators/web-stories-wp
# Improve link handling

## Summary
- Ensures that if `url` is not set, the link is not displayed either.
- Potentially improves the issue of link disappearing.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
Not sure since we don't have a reliable way to reproduce this.

Basic testing would be:
1. Add an element with link and description.
2. Unselect the element and verify it displays the link when hovering over it
3. Select the element and remove the link.
4. Verify the description disappears as well and the element doesn't show link on hovering anymore when unselected.


---



Fixes #4582 (hopefully)
